pyaedt/edb_core/components.py

```diff
@@ -386,7 +386,7 @@ def get_components_from_nets(self, netlist=None):
         return cmp_list
 
     @aedt_exception_handler
-    def create_component_from_pins(self, pins, component_name):
+    def create_component_from_pins(self, pins, component_name, partname, placement_layer=None):
```

Comment:
Please document these new parameters.

Reply:
fixed. Partname removed as not used anymore. Added placement layer parameter doc.